### PR TITLE
restructure active wildfire map

### PR DESCRIPTION
**Pull request by qqnluaq**
wfnews-314, 275, 330, 276,

### CODE REVIEW

**Review by ssylver93 — APPROVED** [review]

**Review by yzlucas — APPROVED** [review]

File: client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    ) {
+        this.incidentsServiceUrl = this.appConfig.getConfig().rest['newsLocal']
+        // console.log(this.incidentsServiceUrl)
+    }
+
+    ngOnInit() {
+
+        this.appConfig.configEmitter.subscribe((config) => {
+            let mapConfig = []
+            // this.checkMapServiceStatus()
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts:54]: If we don't actually need the commented out code, we should remove it

File: client/wfnews-war/src/main/angular/src/app/components/base/base.component.ts [hunk 2]
```diff
@@ -101,9 +101,9 @@ export class BaseComponent implements OnInit, OnChanges, AfterViewInit {
     }
 
     initializeConnectionService() {
-        this.connectionService.monitor().subscribe(isConnected => {
-            this.isConnected = isConnected.hasNetworkConnection;
-        });
+        // this.connectionService.monitor().subscribe(isConnected => {
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/base/base.component.ts:104]: If we don't need it, remove it

File: client/wfnews-war/src/main/angular/src/app/components/wf-map-container/wf-map-container.component.scss [hunk 3]
```diff
@@ -68,83 +84,83 @@
...
-                    align-items: center;
-
-                    i.material-icons {
-                        font-size: 20px;
-                    }
-                }
-            }
-        }
-    }
+    // .smk-bespoke-panel .smk-body .smk-bespoke-content {
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/wf-map-container/wf-map-container.component.scss:87]: If we don't need it, remove it

File: client/wfnews-war/src/main/angular/src/app/services/map-config.service/layers/index.ts [hunk 4]
```diff
@@ -1,34 +1,83 @@
...
+		...FirePerimetersLayerConfig( ls ),
+		// ...SmokeForecastLayerConfig( ls ),
+		...PrescribedFireLayerConfig( ls ),
+		// ...WeatherLayerConfig( ls ),
+        // ...WeatherStationsLayerConfig( ls ),
+        // ...PrecipitationLayerConfig( ls )
+	]
 }
+
+// export function notificationDetailLayerConfig( res: AppResourcesConfig ) {
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/services/map-config.service/layers/index.ts:50]: If we don't need it, remove it

**Review by dhlevi — APPROVED** [review]
There's quite a bit of commented out blocks. If we don't need them, we should remove them so we don't have clutter in the code. If they're needed later, just add a comment indicating why it's being left in

Otherwise, changes look good. We can address cleanup in a later PR due to the volume of changes here.

**Pull request merged**